File: vSongApp/src/main/java/com/jackson_siro/visongbook/models/Callback/CallbackCategory.java
package com.jackson_siro.visongbook.models.Callback;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

import com.jackson_siro.visongbook.models.CategoryModel;

public class CallbackCategory implements Serializable {
    public List<CategoryModel> data = new ArrayList<>();
}

File: vSongApp/src/main/java/com/jackson_siro/visongbook/models/callback/CallbackCategory.kt
package com.jackson_siro.visongbook.models.callback

import java.util.ArrayList
import com.jackson_siro.visongbook.models.CategoryModel
import java.io.Serializable

class CallbackCategory : Serializable {
    lateinit var data: MutableList<CategoryModel>
    //var data: List<CategoryModel> = ArrayList()
}
